### DISCUSSION TITLE
read comment from ply using open3d.io.read_point_cloud

### DISCUSSION THREAD

**Discussion by MeyerBuaharon** [isl-org/Open3D#3952] [Q&A] [answered]
Hey,
I want to get the comments from the ply header.
Right now I'm doing it like this:
```
    ply_data = PlyData.read('./path')
    print('comment:', ply_data.comments)
```
But I want to remove some of the code I write because line after that is followed with this line:
```
        pcd = o3d.io.read_point_cloud('./path', format='ply')
        widget.scene.add_geometry('pcd', pcd, mat)
 ```
 So I looked more into the read_point_cloud, but I can't find a way to get the comments from the ply using this method.
 Is there a way to get the comment with the `read_point_cloud` method?

**Comment by errissa** [comment, marked as answer]
@MeyerBuaharon No, unfortunately you cannot access the comments. The Open3D IO functions are intended to facilitate loading data for further processing or visualization. They are not full fledged parsers for the various file formats.
